### DISCUSSION TITLE
Customise Subform Views - e.g. table format

### DISCUSSION THREAD

**Discussion by thedoctor0124** [Laravel-Backpack/community-forum#1209] [Q&A (Help)] [answered]
The demo site has a very clean and simple Invoice edit page. The subform is laid out like a table. I can't find any documentation to replication this for my project? This is what I have so far:
```
        CRUD::field('colors')->type('relationship')
        ->label('Colors')
        ->subfields([
            [
                'name' => 'name',
                'type' => 'text',
                'label' => 'Color Name',
            ],
            [
                'name' => 'sku',
                'type' => 'text',
                'label' => 'SKU',
            ],
            [
                'name' => 'image',
                'type' => 'image',
                'label' => 'Image',
            ],
            [
                'name' => 'is_active',
                'type' => 'boolean',
                'label' => 'Active',
            ],
        ])
        ->inlineCreate(['entity' => 'color']);
```
And this is the style I want (or as an actual table to save repeating headings would be even better):
![image](https://github.com/user-attachments/assets/7aa278c8-1390-463f-8ee0-c0dd6c0edd69)

**Comment by jcastroa87** [comment, marked as answer]
Hello @thedoctor0124 

You are looking for [this](https://github.com/Laravel-Backpack/demo/blob/main/app/Http/Controllers/Admin/PetShop/InvoiceCrudController.php) Controller?

Cheers.
